portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +23,10 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Dhruvi!</h1>");
-  }
+    
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html;");
+        response.getWriter().println("<h1>Hello Dhruvi!!</h1>");
+    }
```

Comment:
Up to you if you want to use +2 or +4 for indentation. Google's style guide recommends 2 and I would expect all the sample code will be written with 2.

Reply:
tip: I don't know what editor you use, but it's very easy to set indentation by default, and for different file types. Like Kevin said, if the preference for Java is 2, you can set it to 2.